.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  pull_request: # all pull requests
+  push:
+    branches:
+      - master
+
+jobs:
+  hotspot:
+    strategy:
+      matrix:
+        java: [ '8' ] # start slow, then add: , '9', '10', '11', '14'
+    runs-on: 'ubuntu-latest'
+    env:
+      MVN_CMD: ./mvnw --no-transfer-progress -B
+    name: java-${{ matrix.java }}
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - name: Set up JDK
+        uses: actions/setup-java@v2.4.0
+        with:
+          java-version: ${{ matrix.java }}
+          distribution: 'temurin'
+          cache: 'maven'
+      - name: Build
+        run: ${{env.MVN_CMD}} verify
+      - name: Code coverage
+        if:  ${{matrix.java}} == '8'
+        run: ${{env.MVN_CMD}} clean clover:setup test && ${{env.MVN_CMD}} -pl . clover:clover clover:check coveralls:report;
+
+  java-7:
+    runs-on: 'ubuntu-latest'
+    env:
+      MVN_CMD: ./mvnw --no-transfer-progress -B
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - run: |
+          download_url="https://238dj3282as03k369.s3-us-west-1.amazonaws.com/jdk-7u80-linux-x64.tar.gz"
+          wget -O $RUNNER_TEMP/java_package.tar.gz $download_url
+      - name: Set up JDK
+        uses: actions/setup-java@v2.4.0
+        with:
+          distribution: 'jdkfile'
+          jdkFile: ${{ runner.temp }}/java_package.tar.gz
+          java-version: '7'
+          architecture: x64
+          cache: 'maven'
+      - name: "Download and install JCE Unlimited Strength Crypto policies for Oracle JDK 7"
+        run: |
+          curl -q -L -C - https://238dj3282as03k369.s3-us-west-1.amazonaws.com/UnlimitedJCEPolicyJDK7.zip -o $RUNNER_TEMP/UnlimitedJCEPolicyJDK7.zip
+          unzip -oj -d "$JAVA_HOME/jre/lib/security" $RUNNER_TEMP/UnlimitedJCEPolicyJDK7.zip \*/\*.jar
+          rm $RUNNER_TEMP/UnlimitedJCEPolicyJDK7.zip
+      - name: Build
+        run: ${{env.MVN_CMD}} verify
```

Comment:
Also, to ensure coverage checks and reporting, we need from the travis.yml file somewhere:

https://github.com/jwtk/jjwt/blob/2fb6d6bb9c436a27ba109343b26d2d92ced091cb/.travis.yml#L46-L48

Reply:
Oops - sorry - just now saw that you had this in the JDK 8 build